### ISSUE TITLE
Log window should not be an modal window

### ISSUE THREAD

**Issue by hosy** [deszip/BitBot#32]
The log windows has a modal state and is above all windows. It would be great if it has a normal window state to bring this window into the background, behind other windows.

**Comment by deszip** [comment]
Yep, macOS window levels is a mess :( I'll try to fix it one more time on BigSur.
